src/browsers/mswebview2/Ms_Webview2.cc

```diff
@@ -93,7 +93,7 @@ void Ms_Webview2::set_browser_core(browser_core_t &core) {
 
   browser_ready_signal.emit();
 }
-

```

Comment:
@Zacharymk1213
[As before](https://github.com/citkane/gtk-browser-widget/pull/12/files#r2216300674), should be implemented as a virtual method from `gbw::gtk::Browser_window`.

This method is super critical as it is responsible for positioning the `gbw::gtk::Browser_window` over the `gbw::Browser_widget`. Without it the whole product falls down.

It is called on the GTK event loop at `Glib::signal_idle`, so can be resource intensive and impact the GUI in bad ways. Things to look at:
- Is there work we can offload to a worker thread without negatively impacting GTK?
- Do we need to call it on every loop? Maybe a small timeout between 10 -100ms can still give the illusion of a smooth UI update?